src/main.rs

```diff
@@ -96,27 +118,20 @@ struct Opt {
             - *ade* for the ActiveData-ETL specific format;\n\
             - *files* to only return a list of files.\n\
             - *markdown* for human easy read.\n\
+            - *cobertura* for output in coberura format.\n\
```

Comment:
There's a typo, `coberura` should be `cobertura`

Reply:
fixed